src/geometry.cpp

```diff
@@ -79,4 +81,25 @@ namespace geometry {
 	  return v;
 
 	}
+
+	std::vector<Cell> raytraceLine_conversion(const Cell& start, const Cell& end) {
```

Comment:
I've never tried this myself, but can you try adding `[[no discard]]` to this https://en.cppreference.com/w/cpp/language/attributes/nodiscard

Reply:
Done

Reply:
Done